R/plot-individual-time-profile.R

```diff
@@ -53,13 +53,16 @@ plotIndividualTimeProfile <- function(dataCombined,
   individualTimeProfilePlotConfiguration$export <- defaultInternalPlotConfiguration$export
 
   # If axes labels haven't been specified, create them using dimensions and units.
+  xUnitString <- ifelse(unique(df$xUnit) == "", unique(df$xUnit), paste0(" [", unique(df$xUnit), "]"))
```

Comment:
probably you can define
xUnitString <- unique(df$xUnit);
and then 
 xUnitString <- ifelse(xUnitString  == "", xUnitString , paste0(" [",xUnitString , "]"))

Reply:
Thanks, using this code now.